vng/metadata.go

```diff
@@ -294,10 +294,20 @@ func metadataValue(zctx *super.Context, b *zcode.Builder, m Metadata) super.Type
 		}
 		b.EndContainer()
 		return zctx.MustLookupTypeRecord(fields)
+	case *Named:
+		return metadataValue(zctx, b, m.Values)
```

Comment:
Shouldn't need this case since there's a call to Under in the switch expression.
```suggestion
```

Reply:
Indeed